### PR TITLE
fix: device arg not work, rename device to ngpu

### PR DESCRIPTION
**Pull request by veelion**

### CODE REVIEW

File: modelscope/pipelines/builder.py [hunk 1]
```diff
@@ -128,7 +128,11 @@ def pipeline(task: str = None,
             pipeline_props = {'type': pipeline_name}
 
     pipeline_props['model'] = model
-    pipeline_props['device'] = device
+    # 'device' not in 'AutomaticSpeechRecognitionPipeline.get_cmd()', but ngpu
```
Comment (wenmengzhou) [inline modelscope/pipelines/builder.py:131]: this logic shold be added in the __init__ function of AutomaticSpeechRecognitionPipeline,  other pipeline use the device gpu/cpu as expected values

**Comment by yeates** [comment]
Then how to select a single gpu to pipeline in this way? like using arg `device='cuda:3'` seems can not work

**Comment by veelion** [comment]
> Then how to select a single gpu to pipeline in this way? like using arg `device='cuda:3'` seems can not work. Actually, when I run the code as follows, `[pipeline('text-to-video-synthesis', 'damo/text-to-video-synthesis', device=f"cuda:3")` this pipeline still load on cuda:0, not cuda:3, that's weird.

Yes, that's weird, `ngpu` in `AutomaticSpeechRecognitionPipeline` means `how many GPUs to use`,  acturally only one to use:

https://github.com/alibaba-damo-academy/FunASR/blob/c5c944eed9b11b9ec1cd8f0ec50d235c33fe161e/funasr/bin/asr_inference_paraformer.py#L618


`cuda:3` wants to use `the 3rd GPU`, if this need to be supported, maybe it's need to re-design the usage of `ngpu`

**Review by langgz — APPROVED** [review]
# pipeline() from pipelines/builder.py passes 'device' but 'ngpu' needed here
        device = extra_args.get('device')
        if device == 'cpu':
            extra_args['ngpu'] = 0

ngpu: 1(Defautl), gpu is used if gpu is available

**Review by tastelikefeet — APPROVED** [review]

**Pull request merged**